### PR TITLE
Version/3 picture fill symbol

### LINKED ISSUE #297: Implement PictureFillSymbol
https://developers.arcgis.com/javascript/latest/api-reference/esri-symbols-PictureFillSymbol.html

### PR DESCRIPTION
**Pull request by TimPurdum**
closes #297

### CODE REVIEW

File: src/dymaptic.GeoBlazor.Core/GeoBlazorSettings.cs [hunk 1]
```diff
@@ -14,15 +14,4 @@ public class GeoBlazorSettings
     ///     The GeoBlazor Pro License Key, generated on the licensing server.
     /// </summary>
     public string? LicenseKey { get; set; }
-    
-    /// <summary>
-    ///     The Application Name for a MAUI application. Not required for Blazor Server or Blazor WebAssembly.
-    /// </summary>
-    public string? MauiAppName { get; set; }
```
Comment (seahro) [inline src/dymaptic.GeoBlazor.Core/GeoBlazorSettings.cs:21]: seems unrelated, did you mean to remove these?

**Review by seahro — APPROVED** [review]
looks good.  one question on the changes in `GeoBlazorSettings.cs`

File: src/dymaptic.GeoBlazor.Core/GeoBlazorSettings.cs (hunk 1 above)
Comment (TimPurdum) [reply src/dymaptic.GeoBlazor.Core/GeoBlazorSettings.cs:21]: looks like I got a new merge from `develop` mixed in here that wasn't already merged to v3, but yes, this is going away.
